test-app/types/ember-data/types/registries/model.d.ts

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
I don't think this is needed as we don't use ember-data.

Reply:
aggr.. added by blueprint and i have forgot to remove... sry